### ISSUE TITLE
为什么不在每个训练epoch中更新pseduo label ?

### ISSUE THREAD

**Issue by Taotaoxu** [weijiawu/Synthetic-to-Real-Unsupervised-Domain-Adaptation-for-Scene-Text-Detection-in-the-Wild#8]
为什么不在每个训练epoch中更新pseduo label ?

https://github.com/weijiawu/SyntoReal_STD/blob/02ee312504d754bf51d5149916235110d5283105/trainSyndataToICDAR15.py#L138-L143

可以改成以下这样吗？

```
for epoch in range(args.epoch_iter):
    generate_pseduo(model, args.target_image, args.target_pseudo_positive, args.target_pseudo_negative, device)
    train(epoch, model, optimizer, train_loader_source, train_loader_target)
    
```

**Comment by weijiawu** [comment]
可以的，可以改成这个样子训练，迭代更新

**Comment by entertainment-huren** [comment]
where is 'net work/loss_target.py' file .请问你有这一个文件可以给我吗，这对我非常重要，非常感谢！！！Do you have this one document for me, which is very important to me, thank you very much
